docs/src/main/paradox/persistence-query.md

```diff
@@ -33,9 +33,6 @@ recommend (in the spirit of CQRS) of splitting up the write/read sides into sepa
 For a similar implementation of query interface to @ref:[Durable State Behaviors](typed/durable-state/persistence.md)
 please refer to @ref:[Persistence Query using Durable State](durable-state/persistence-query.md).
 
-The @extref[Microservices with Pekko tutorial](platform-guide:microservices-tutorial/) explains how to
-implement an Event Sourced CQRS application with Pekko Persistence and Pekko Projections.
-
```

Comment:
How about leaving a 

>//TODO add platform-guide 

Otherwise lgtm

Reply:
to me, it's easier to have a git issue and the git commit id that needs to be reverted